specifications/sep-payment-channel-transactions.md

```diff
@@ -170,9 +170,11 @@ without any payment.
 7. I and R sign and exchange the formation transaction F.
 8. I or R submit F.
 
-Participants should defer deposits of initial contributions till after formation
-for channels that will hold trustlines to issuers that are not auth immutable,
-and could be clawback enabled. See [Security](#Security).
+Participants may prefer to make deposits of initial contributions after
+formation if the channel will hold a trustline to an issuer that is not auth
+immutable, the participants' trustlines are not clawback enabled, and the
+participants do not want the trustlines of the escrow accounts to be clawback
+enabled. See [Security](#Security).
 
```

Comment:
I'm having trouble wrapping my head around this section, I think bc I don't fully understand clawback.

❓ why does it matter if the initial contribution is made after formation if the asset is a clawback enabled one?

Reply:
I think I might be understanding my confusion. Should this be instead:
```
the participants' trustlines are clawback enabled, or the
participants do not want the trustlines of the escrow accounts to be clawback
enabled.
```

Reply:
Yeah this is not particularly easy to read, but I think it is accurate. Let's see if we can make it clearer.

Clawback can be enabled on an issuer account at anytime, unless the issuer account is "auth immutable". When a trust line is created the trust line inherits the clawback enabled flag from the issuer account. This means that when the escrow account adds its trust line is the point in time that we can have confidence about whether the escrow account's trust line will be clawback enabled or not.

If the initial contribution is made before formation and the other participants escrow account's trust line is created during formation you don't know if the other escrow account's trust line will be clawback enabled. All of this is probably fine for the average user, but if a user specifically doesn't want for their assets, and their payment channel to be subject to the clawback enabled capability, they'd ideally want to see the formation take place and ensure the other escrow account is not clawback enabled before depositing any asset to the channel.

Writing all that out I it does occur to me now that this is a bit of a special case and esoteric. It applies to most assets – because most assets are not auth immutable – but it only applies to users who are particularly sensitive to clawback enabled.

I think maybe we should reduce this paragraph to simply saying that in some cases users wanting to avoid their asset and channel being subject to the clawback enabled capability should read the security section. In that section we can discuss it in more detail.

Wdyt? If you think that sounds good I can take a stab at updating this PR.

Reply:
gotchya thank you for the explanation! I understand now. 

Ya I agree with you about putting the bulk of the clawback explanation in the security section. I think the extra space to explain this case fully is helpful (at least it took me a bit to wrap my head around what this paragraph was saying). Suggesting below for the rework:

```
Participants may prefer to make deposits of initial contributions after
formation to ensure clawback trustlines are set correctly. See [Security](#Security).
```
and then in the security section it's explained in more detail.